src/classes/HH_Container_LCTRL.cls

```diff
@@ -37,6 +37,28 @@
 public class HH_Container_LCTRL {
     
     /*******************************************************************************************************
+    * @description trick the packaging spider to pickup references to custom labels only used by our
+    * lightning components.
+    */
+    public static void packageLabels() {
```

Comment:
I only include those labels that reside in the npsp package.  the other labels are in the npo02 package, and including them here will do nothing, since they don't live in our npsp package.